libs/eavmlib/src/atomvm.erl

```diff
@@ -267,3 +268,23 @@ posix_read(_File, _Count) ->
     {ok, non_neg_integer()} | {error, posix_error()}.
 posix_write(_File, _Data) ->
     erlang:nif_error(undefined).
+
+%%
+%% @param   Unit The time unit
+%% @param   ValueSinceUnixEpoch The value, in specified units, since the UNIX epoch (Jan 1, 1970)
+%% @return  `ok'
+%% @doc Set the system time.
+%%
+%% This function sets the system time to the specified number of units
+%% (in seconds, milliseconds, or microseconds) since the UNIX epoch (Jan 1, 1970).
+%% Coordinates are all in UTC.
+%%
+%% Note.  Some systems may require special permissions to call this function.
+%% @end
+%%
+-spec posix_clock_settime(
+    Unit :: second | millisecond | microsecond, ValueSinceUnixEpoch :: integer()
+) ->
+    ok.
```

Comment:
Return should be ok | error.

Reply:
I don’t think there is a path in this function for returning an error tuple.  See the implementation below.